src/styles/styles.js

```diff
@@ -228,23 +228,18 @@ const styles = {
     pill: {
         borderRadius: 14,
         backgroundColor: themeColors.pillBG,
```

Comment:
I think we'll also have a gray badge that will be the default case, so we might want to think about how to manage the styles here. Maybe we have `badge` as well as `badge.success` or something? Similar to how we handle default buttons versus success buttons? 

Or the default badge is green, and then we can also have `badge.neutral` or something?